### ISSUE TITLE
Secret channel automation

### ISSUE THREAD

**Issue by pokorj54** [fit-ctu-discord/honza-botner#108]
Auto assign roles for certain channel - "the real channel"

**Comment by tenhobi** [comment]
We can check if the message contains the regex expression from some secret config.

**Comment by ostorc** [comment]
Yes, we will have to go that way. Also since it is supposed to be top secret we will have to store it in Heroku config. Also this issue will have to wait for #83
